src/ua/dymohlo/geometricshapes/Triangle.java

```diff
@@ -0,0 +1,32 @@
+package ua.dymohlo.geometricshapes;
+/*
+Class Triangle has to has 3 overloading area methods :
+	area of triangle using Heron's formula
+	area of triangle using the product of two sides per sine angle
+	area using a base height product
+*/
+
+public class Triangle extends Figure {
+    double sideA = 2;
```

Comment:
Please, use constructor to initialize variables
For the record: class fields should be private